rest_framework/relations.py

```diff
@@ -58,6 +60,16 @@ def __init__(self, *args, **kwargs):
 
     def initialize(self, parent, field_name):
         super(RelatedField, self).initialize(parent, field_name)
+
+        if self.queryset is None and not self.limit_choices_to:
+            manager = getattr(self.parent.opts.model, self.source or field_name)
+            try:
+                self.limit_choices_to = manager.field.rel.limit_choices_to
+            except AttributeError:
+                # GenericForeignKey and their reverse relationships don't have
+                # a `field` property.
+                pass
+
```

Comment:
In the event that the model field is defined as:

```
class MyModel(models.Model):
    user = models.ForeignKey(User, limit_choices_to={'is_active': True})
```

it seemed appropriate for DRF to respect that value and filter the default queryset.

Do you think it would be better to only do filtering when `limit_choices_to` is explicitly declared on the serializer field?